client/src/components/Workbench/Workbench.js

```diff
@@ -33,6 +33,8 @@ function Workbench() {
     shallowEqual
   );
 
+  const hideApps = useSelector(state => state.workbench.config.hideApps);
```

Comment:
We could combine this `useSelector` call with our other calls above on [line 25](https://github.com/TACC/Core-Portal/pull/447/files#diff-a04c067eab18809c9a8dd77b563de39c277d430ea1de1b8c7a914d3c8f98b45aR25-R32)